.changeset/purple-flies-end.md

```diff
@@ -0,0 +1,23 @@
+---
+'@razorpay/blade': minor
+---
+
+**feat: Added internationalization in Amount component via i18nify.
+References**
+
+- **i18nify-js:** https://www.npmjs.com/package/@razorpay/i18nify-js
+- **i18nify-react:** https://www.npmjs.com/package/@razorpay/i18nify-react
+
+**What changes ?**
+
+1. The `<Amount />` component will now automatically format numbers based on the user's browser locale. For example, `<Amount value={123456.789} currency="INR">` will render `₹1,23,456.79` for browsers with the `en-IN` default locale, whereas it will render `₹123,456.79` for browsers with the `en-US` locale.
+
+2. If you want to enable users to change the locale of your page, add the `@razorpay/i18nify-react` package and wrap your app inside the `I18nProvider`. Utilize the `setI18nState` utility to modify the locale. For more details, please refer to the [documentation](https://www.npmjs.com/package/@razorpay/i18nify-react).
+
+3. Additionally, if you prefer to maintain a fixed locale for your page and amount component, enclose your app within `<I18nProvider initData={{locale: 'locale-you-want'}}>..`. For more details, please refer to the [documentation](https://www.npmjs.com/package/@razorpay/i18nify-react).
+
+**How to update ?**
```

Comment:
we shall also update the [installation guide](https://blade.razorpay.com/?path=/docs/guides-installation--docs) with installation of i18nify. its located at `packages/blade/docs/guides/Installation.stories.mdx`

Reply:
Sure, I was planning to send this in a separate PR, but we can incorporate these changes into this one. 
I'll need your assist to finalize the content and format for inclusion here. @kamleshchandnani

Reply:
updated.